lib/consult/template.rb

```diff
@@ -21,12 +21,13 @@ def render(save: true)
       renderer = ERB.new(contents, nil, '-')
       result = renderer.result(binding)
 
+      puts "Consult: rendering #{name}..." if verbose?
```

Comment:
To be consistent with the other logging messages, I think we should capitalize the first word after `Consult: `, e.g.

```suggestion
      puts "Consult: Rendering #{name}..." if verbose?
```

Reply:
While we're here in the name of CLI improvements, would it be useful to include the path of where it is rendered?

For example:

```suggestion
      puts "Consult: rendering #{name} to #{path}..." if verbose?
```

Example output:

```
$ bundle exec consult
Consult: Loading config from /Users/henry.thach/project/config/consult.yml
Consult: rendering foo to /Users/henry.thach/project/config/some/path/to/foo.yml...
Consult: rendering bar to /Users/henry.thach/project/config/bar.yml...
Consult: rendering baz to /Users/henry.thach/project/whatever/baz.yml...
```

Reply:
Ha, that's exactly how I had it in an earlier iteration. I reverted it because it was a bit too verbose but I'll restore it now with your vote

Reply:
Just had a chance to test. I agree that showing the path of the rendered file is helpful.